variables.tf

```diff
@@ -1 +1,94 @@
-# Variables placeholder
+variable "protect_s3_bucket_resources" {
+  description = "S3 bucket resource ARNs to protect from bucket and object deletion"
+  type        = list(string)
+  default     = [""]
+}
+
+variable "protect_iam_role_resources" {
+  description = "IAM role resource ARNs to protect from modification and deletion"
+  type        = list(string)
+  default     = []
+}
+
+variable "allowed_regions" {
+  description = "AWS Regions allowed for use (for use with the restrict regions SCP)"
+  type        = list(string)
+  default     = [""]
+}
+
+variable "target" {
+  description = "id of OU or account on which to attach SCP"
+  type        = string
+}
+
+variable "add_deny_root_account" {
```

Comment:
I do not like the name of these variables -- the `add` part adds nothing, and I think it makes things more confusing. Just drop it from everything.

Reply:
yeah this was just so I could get my head around `allow_deny`. I realize it's less than ideal.